package.json

```diff
@@ -102,7 +102,7 @@
     "@types/puppeteer": "^1.11.1",
     "@types/react": "~16.7.22",
     "@types/react-dom": "~16.0.11",
-    "@types/react-is": "^16.7.1",
+    "@types/react-is": "~16.5.0",
```

Comment:
@moog16 I'm rolling this back to one released at the same time as `@types/react ~16.7.22` due to conflicts.